chart/templates/rbac/service_account.yaml

```diff
@@ -8,3 +8,10 @@ metadata:
     app.kubernetes.io/managed-by: helm
   name: {{ .Values.serviceAccountName }}
   namespace: {{ .Release.Namespace }}
+  {{- if .Values.imagePullSecrets }}
+imagePullSecrets:
+  {{- range .Values.imagePullSecrets }}
+  - name: {{ . }}
+  {{- end }}
+  {{- end }}
+  
```

Comment:
strange. it still complains about missing newline. I guess it's not that big of a deal